android/src/main/java/com/cmcewen/blurview/BlurViewPackage.java

```diff
@@ -20,11 +20,13 @@
         return modules;
     }
 
-    @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-        return Collections.emptyList();
-    }
-
+   // if react native < 47, open code
+   /* @Override
```

Comment:
all one needs to do is comment out just `@Override` statement instead of the entire method. Reason being  purely backward compatibility